doc/rpc/pathfinder_rpc_api.json

```diff
@@ -267,61 +267,6 @@
                     "ABORTED"
                 ],
                 "description": "The status of a transaction"
-            },
-            "BLOCK_HEADER": {
-                "type": "object",
-                "properties": {
-                    "hash": {
-                        "ref": "#/components/schemas/FELT"
-                    },
-                    "parent_hash": {
-                        "ref": "#/components/schemas/FELT"
-                    },
-                    "number": {
-                        "type": "integer"
-                    },
-                    "timestamp": {
-                        "type": "integer"
-                    },
-                    "gas_price": {
-                        "ref": "#/components/schemas/FELT"
-                    },
-                    "sequencer_address": {
-                        "ref": "#/components/schemas/FELT"
-                    },
-                    "starknet_version": {
-                        "ref": "string"
-                    },
-                    "class_commitment": {
-                        "ref": "#/components/schemas/FELT"
-                    },
-                    "event_commitment": {
```

Comment:
Is this really supposed to be removed?

Reply:
Yes it moved to the other spec file; it's only used in the subscription api.